R/dvhydrograph-render.R

```diff
@@ -287,6 +300,11 @@ getDVHydrographPlotConfig <- function(plotItem, plotItemName, yLabel="", minMaxE
                            lty=ifelse(plotItem[['newSet']] == "est", 1, 6), col=ifelse(plotItem[['newSet']] == "est", "orange", "red3")),
                       styles$est_lines)
     ),
+    estimated4Edges = list(
+      arrows = append(list(x0=plotItem[['time']], x1=plotItem[['time']], y0=plotItem[['y0']], y1=plotItem[['y1']], 
+                           lty=ifelse(plotItem[['newSet']] == "est", 1, 7), col=ifelse(plotItem[['newSet']] == "est", "black", "red3")),
```

Comment:
It's almost like `plotItemName` needs to be changed to `estimated1EdgesEst` or `estimated1EdgesStat` instead of just `estimated1Edges` prior to entering this function. It seems like the file grabbing the configs shouldn't have to deal with that kind of logic - it should happen upstream.

This might be getting to be a bigger task to clean that up. Could be a new ticket instead.